### ISSUE TITLE
Links in text should have a 3:1 contrast with surrounding text OR be underlined

### ISSUE THREAD

**Issue by helrond** [RockefellerArchiveCenter/rockarch.org#57]

**Comment by HaSistrunk** [comment]
https://webaim.org/resources/linkcontrastchecker/
